include/audio/sounds.h

```diff
@@ -19,6 +20,8 @@ namespace audio {
 AudioControlSGTL5000 sgtl5000_1;
 
 void InitAudio() {
+  
+  Serial.begin(9600);

```

Comment:
Can we remove this? `Serial` refers to the Debug USB Serial. We should rely on external code to deal with Debug initialization and usage. Also, I think we can remove `#include <SerialFlash.h>` above